### PR TITLE
esbuild that has scss plugin installed

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -126,13 +127,13 @@
             <configuration>
               <target>
                 <get dest="src/main/resources" skipexisting="true">
-                  <url url="https://registry.npmjs.org/@esbuild/darwin-arm64/-/darwin-arm64-${esbuild.version}.tgz"/>
```
Comment (maxandersen) [inline pom.xml:129]: is there no other way to build this than depend on a personal github account releases ? or is this just a draft/experiment?

Comment (phillip-kruger) [reply pom.xml:129]: Probably we should move that to the mvnpm namespace

Comment (phillip-kruger) [reply pom.xml]: Yes, that is the plan, until (if ever) it is supported in esbuild. @edewit did you do a PR against esbuild ? Or are they not interested in this ?

**Comment by ia3andy** [comment]
Thanks for this @edewit this is awesome!

I wasn't able to work in the past few days and I have a stack on my todo list. I will have a closer look when things settle. In the mean time could you transfer the artifacts to the mvnpm org?

File: pom.xml (hunk 1 above)
Comment (edewit) [reply pom.xml]: I have @phillip-kruger added a PR that will introduce an API to be able to add plugins to the cli, but we will still need to build or own version with the plugins that we want to use.

**Comment by edewit** [comment]
@ia3andy @maxandersen moved to mvnpm/esbuild

**Comment by maxandersen** [comment]
moved to https://github.com/evanw/esbuild/pull/3539

**Review by ia3andy — APPROVED** [review]
Thanks @edewit, this is awesome!!

**Pull request merged**